feature-engineering/tokenizer.md

```diff
@@ -0,0 +1,30 @@
+Transform an SFrame into the Penn-Tree-Bank-style tokenization.
+
+For an SFrame of strings, where each row is assumed to be a natural English language document, the tokenizer transforms each row into an ordered list of strings that represents the Penn-Tree-Bank-style tokenization of that row's document.
```

Comment:
It would also be useful to add a brief description of what this is and how it works. The description as is sells the feature short of how useful it is.